### PR TITLE
soapyrtlsdr 0.3.2 (new formula)

### PR DESCRIPTION
**Pull request by NeuroForLunch**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

This formula depends on soapysdr which is another new formula I submitted and is waiting for approval.

### CODE REVIEW

**Comment by NeuroForLunch** [comment]
Not sure what tests to add to this or if they are needed.

File: Formula/soapyrtlsdr.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Soapyrtlsdr < Formula
+  desc "SoapySDR RTL-SDR Support Module"
+  homepage "https://github.com/pothosware/SoapyRTLSDR/wiki"
+  url "https://github.com/pothosware/SoapyRTLSDR/archive/soapy-rtl-sdr-0.3.2.tar.gz"
+  sha256 "d0335684179d5b0357213cc786a78d7b6dc5728de7af9dcbf6364b17e62cef02"
+  license "MIT"
+  revision 1
```
Comment (carlocab) [inline Formula/soapyrtlsdr.rb:7]:
```suggestion
```

File: Formula/soapyrtlsdr.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "true"
```
Comment (carlocab) [inline Formula/soapyrtlsdr.rb:22]: This needs a proper test.

File: Formula/soapyrtlsdr.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  desc "SoapySDR RTL-SDR Support Module"
+  homepage "https://github.com/pothosware/SoapyRTLSDR/wiki"
+  url "https://github.com/pothosware/SoapyRTLSDR/archive/soapy-rtl-sdr-0.3.2.tar.gz"
+  sha256 "d0335684179d5b0357213cc786a78d7b6dc5728de7af9dcbf6364b17e62cef02"
+  license "MIT"
+  head "https://github.com/pothosware/SoapyRTLSDR.git"
+
+  depends_on "cmake" => :build
+  depends_on "soapysdr"
+  depends_on "librtlsdr"
```
Comment (carlocab) [inline Formula/soapyrtlsdr.rb:11]:
```suggestion
  depends_on "librtlsdr"
  depends_on "soapysdr"
```

**Comment by NeuroForLunch** [comment]
The tests are failing because its attempting to run from the soapyrtlsdr folder when it should run from the soapysdr folder and check if soapyrtlsdr is there.

File: Formula/soapyrtlsdr.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    assert_match "Checking driver 'rtlsdr'... PRESENT", shell_output("#{bin}/SoapySDRUtil --check=rtlsdr")
```
Comment (carlocab) [inline Formula/soapyrtlsdr.rb:21]:
```suggestion
    assert_match "Checking driver 'rtlsdr'... PRESENT",
                 shell_output("#{Formula["soapysdr"].bin}/SoapySDRUtil --check=rtlsdr")
```

File: Formula/soapyrtlsdr.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    assert_match "Checking driver 'rtlsdr'... PRESENT",
+             shell_output("#{Formula["soapysdr"].bin}/SoapySDRUtil --check=rtlsdr")
```
Comment (carlocab) [inline Formula/soapyrtlsdr.rb:22]:
```suggestion
                shell_output("#{Formula["soapysdr"].bin}/SoapySDRUtil --check=rtlsdr")
```
Let's align the arguments to `assert_match` here.

**Review by carlocab — APPROVED** [review]
Nice work! Thanks for this.

**Pull request closed without merging**